schema/bundle.schema.json

```diff
@@ -286,7 +288,8 @@
                 },
                 "path":{
                     "description": "The path inside of the invocation image where output will be written",
-                    "type": "string"
+                    "type": "string",
+                    "pattern": "^\/cnab\/app\/outputs\/.+$"
```

Comment:
Good. If I'm reading this correctly, it specifies a strict subpath of `/cnab/app/outputs`.

Reply:
Yeah, I added the wording in the prose to clarify this :)